docker/release

```diff
@@ -88,6 +88,7 @@ RUN python -m pip install --prefer-binary --no-cache-dir --upgrade pip==23.1.2 &
     pip install --no-cache-dir 'pysurrealdb' 'websocket' || true && \
     pip install --no-cache-dir 'pygithub' || true && \
     pip install --prefer-binary --no-cache-dir hugging_py_face || true && \
+    pip install --prefer-binary --no-cache-dir together || true && \
```

Comment:
I don't think we'll make this a default package just yet, so please remove this.